src/sheets/storeOrder.js

```diff
@@ -76,7 +76,7 @@ function itemToSheetData(variant, dbId, item, order, userJiraId, date) {
     : {
         Note: order.reason,
         Urgent: Boolean(order.isUrgent),
-        'From invoice': order.paymentType === 'payment_type_invoice',
+        ...(variant === 'wincent' ? {} : {'From invoice': order.paymentType === 'payment_type_invoice'}),
```

Comment:
Same as line 74 - I guess ideally these variant-specific columns would be configured in `constants.js` - not sure if we want to do that now

Reply:
I think it is ok for now. If there are more change requests, we can consider making it nicer :)